models/addresses_events/arbitrum/addresses_events_arbitrum_first_funded_by.sql

```diff
@@ -0,0 +1,44 @@
+{{ config(
+    schema = 'addresses_events_arbitrum'
+    , alias = 'first_funded_by'
+    , materialized = 'incremental'
+    , file_format = 'delta'
+    , incremental_strategy = 'merge'
```

Comment:
since we have an anti-join on the existing table which should already eliminate any duplicates, we can use `append` instead of `merge` here. (same comment on all the other models)